### PR TITLE
feat: add script to configure v3

### PR DESCRIPTION
**Pull request by evan-forbes**
## Overview

adds a script and better docs for configuring everything for v3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -301,6 +301,18 @@ disable-mptcp:
 
 .PHONY: disable-mptcp
 
+CONFIG_FILE ?= ${HOME}/.celestia-app/config/config.toml
+SEND_RECV_RATE ?= 10485760  # 10 MiB
+
+configure-v3:
```
Comment (rach-id) [inline Makefile:307]: is this intended to also work on mac?
If so, I can try to fix it as I am seeing this when executing:
```
Using config file at: /Users/midnight/.celestia-app/config/config.toml
sed: 1: "/Users/xxx/.celest ...": invalid command code m
sed: 1: "/Users/xxx/.celest ...": invalid command code m
sed: 1: "/Users/xxx/.celest ...": invalid command code m
make: *** [configure-v3] Error 1
```

Comment (evan-forbes) [reply Makefile:307]: ahh good catch! do you mind trying this now? https://github.com/celestiaorg/celestia-app/pull/4022/commits/ad3d55d713e1f560952a0e9838fd31465bf67a01

**Review by rach-id — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (rach-id) [reply Makefile:307]: works, thanks 🙏 🙏

**Review by ninabarbakadze — APPROVED** [review]
utAck

**Pull request merged**